packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.scss

```diff
@@ -83,3 +83,6 @@
     height: calc(100vh - 3rem);
   }
 }
+:global(.bx--time-picker), :global(.bx--date-picker-container), :global(.bx--date-picker), :global(#visitStartDateInput) {
+  width: 100%;
+}
```

Comment:
`:global` styles on Carbon classes should go in `esm-styleguide`.

The global style on `#visitStartDateInput` should be changed to a normal scoped class style, i.e.

```css
.visitStartDateInput {
  width: 100%;
}
```